bedrock/firefox/templates/firefox/android/index.html

```diff
@@ -344,7 +344,7 @@ <h2 class="heading">{{ _('Don’t leave home without being in Sync') }}</h2>
 
         <div id="download-content">
           <h3>{{ _('Choose Firefox') }}</h3>
-          <a class="dl-button" rel="external" href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}">
+          <a class="dl-button" rel="external" href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}" data-button-type="download" data-interaction="button download click" data-download-version="Firefox for Android">
```

Comment:
Just double checking that `data-interaction` should be different on the two links changed above. Assuming the second had `button download click` because it's in the "download" section.